### PR TITLE
Validations

### PR DESCRIPTION
**Pull request by BruceDev**
Added integer validations

### CODE REVIEW

**Comment by vlucas** [comment]
Whoops - commented on the wrong PR - didn't notice you closed it and opened this one instead. Please check the comments on #117 - they all still apply to this one.

**Comment by BruceDev** [comment]
Hey thanks for the tips. New to contributing so appreciate you helping me out. I did the changes and tested. It throws the error if its anything BUT solid numbers, so no periods, commas, percentage signs, and plus signs are able to pass by.

File: src/Validator.php [hunk 1]
```diff
@@ -60,6 +60,21 @@ function ($value) {
...
     /**
+     * Assert that each specified variable is a number.
+     *
+     * @return \Dotenv\Validator
+     */
+    public function isInteger()
+    {
+        return $this->assertCallback(
+            function ($value) {
+                return (ctype_digit($value) == true);
```
Comment (vlucas) [inline src/Validator.php]: The `ctype_digit` function already returns a boolean. Change this to `return ctype_digit($value)`.

**Comment by BruceDev** [comment]
Done, thanks for that.

**Comment by GrahamCampbell** [comment]
What's wrong with `is_numeric`?

**Comment by GrahamCampbell** [comment]
We're missing tests.

**Comment by GrahamCampbell** [comment]
I've squashed and merged this. Thanks for the contribution.

**Pull request closed without merging**